### ISSUE TITLE
Saved checkpoints not titled with descriptive information

### ISSUE THREAD

**Issue by nburner96** [amaiya/ktrain#519]
Hello, I am training a ResNet50 model and designating a checkpoint folder to save models along the way with
`learner.fit_onecycle(1e-4, epochs, checkpoint_folder='models')`

However, in the checkpoint folder the saved model names only contain the epoch:

![image](https://github.com/amaiya/ktrain/assets/21323696/b95bba66-4f22-4772-9868-a1dbfe6c6bd5)

instead of including the epoch and validation loss as described in the core.py documentation. Is there a way to fix this?

**Comment by amaiya** [comment]
Thanks  - this is a bug and will be fixed in the next release.
